### PR TITLE
add a dockerfile to the repository

### PR DESCRIPTION
**Pull request by antoine-de**
add a dockerfile

this will makes it possible to have an automated build on hub.docker.com

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+*
+!geocoder_tester
+!Pipfile*
+!run_qwant.sh
+!conftest.py
```
Comment (amatissart) [inline .dockerignore]: These files are explictly ADDed in the Dockerfile.
Why not simply put python-related files here: 
```
__pycache__
*.pyc
*.pyo
*.pyd
.Python
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ADD Pipfile* run_qwant.sh conftest.py /
+
+WORKDIR /
+
+RUN pipenv install --system --deploy
+
+# the sources are copied as late as possible since they are likely to change often
+ADD geocoder_tester/ /geocoder_tester/
+
+CMD ["/run_qwant.sh", "/results", "http://bragi:4000/autocomplete"]
```
Comment (amatissart) [inline Dockerfile]: Maybe we could use an ENV or a build ARG to set the geocoder url ?

File: .dockerignore (hunk 1 above)
Comment (antoine-de) [reply .dockerignore]: I also have lots of tests results in the directory and I'd prefer not to have to include them.

the best way would be to have `!geocoder_tester/**.py` but it does not seems to work

File: Dockerfile [hunk 3]
```diff
@@ -15,4 +15,6 @@ RUN pipenv install --system --deploy
 # the sources are copied as late as possible since they are likely to change often
 ADD geocoder_tester/ /geocoder_tester/
 
-CMD ["/run_qwant.sh", "/results", "http://bragi:4000/autocomplete"]
+ENV BRAGI_URL "http://bragi:4000/autocomplete"
+
+CMD ["/run_qwant.sh", "/results", ${BRAGI_URL}]
```
Comment (amatissart) [inline Dockerfile]: Hum, does that really help to override the URL programmatically ?

Comment (antoine-de) [reply Dockerfile]: yes this way you can do: docker run -it -e BRAGI_URL=bob geocoder_tester

(and there is a default value to "http://bragi:4000/autocomplete"

**Pull request merged**